rdkit/Chem/Draw/MolDrawing.py

```diff
@@ -455,11 +413,10 @@ def AddMol(self, mol, centerIt=True, molTrans=None, drawingTrans=None, highlight
           symbolLength = len(symbol)
         else:
           base = atom.GetSymbol()
-          if (base == 'H' and (iso == 2 or iso == 3) and
-              self.drawingOptions.atomLabelDeuteriumTritium):
-            if (iso == 2):
+          if base == 'H' and self.drawingOptions.atomLabelDeuteriumTritium:
```

Comment:
I don't understand why you made this change and think it makes the code less clear. I'd like this one changed back.